tempesta_fw/apm.c

```diff
@@ -1693,19 +1693,13 @@ tfw_cfgop_apm_server_failover(TfwCfgSpec *cs, TfwCfgEntry *ce)
 			   ce->vals[0]);
 		return -EINVAL;
 	}
-	if (tfw_cfg_parse_int(ce->vals[1], &limit)) {
-		TFW_ERR_NL("Unable to parse http limit value: '%s'\n",
-			   ce->vals[1]);
+	if (tfw_cfg_parse_int(ce->vals[1], &limit))
```

Comment:
I don't really like how Tempesta complains about parsing errors, e.g. for line `server 127.0.0.1:8080 conns_n=1a;`:

On master:
```
[tempesta] ERROR: Invalid value: '1a'
[tempesta] ERROR: configuration parsing error:
   5: server 127.0.0.1:8080 conns_n=1a;
      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

On my branch:
```
[tempesta] ERROR: Can't parse '1a' as 'int'
[tempesta] ERROR: configuration parsing error:
   5: server 127.0.0.1:8080 conns_n=1a;
      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

As for me, last option is more verbose. Yes, that commit made the output less verbose  for some options. But before changes it looked ugly: every directive has it's own unique error message, error processing is also unique for every directive. Some errors described very good, while descriptions for others are completely missed. My variant is much easier to support.

The thing I completely dislike in parsing errors is reversed error message. Ideal message for me is below:
```
[tempesta] ERROR: configuration parsing error:
   5: server 127.0.0.1:8080 conns_n=1a;
                                    ^^
[tempesta] ERROR: Can't parse '1a' as 'int', allowed values are [1, 65535]
```

Reply:
Agree, the change is actually good since we still have
```
   5: server 127.0.0.1:8080 conns_n=1a;
      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```
And yes,
```
   5: server 127.0.0.1:8080 conns_n=1a;
                                    ^^
```
would look much better.